scripts/devenv-builder/create-vm.sh

```diff
@@ -34,31 +34,31 @@ DATAVOLSIZE=${8:-$DATAVOLSIZE}
 [[ ! "${DATAVOLSIZE}" =~ ^[0-9]+$ ]] || [[ "${DATAVOLSIZE}" -le 0 ]] && usage "Invalid data volume size: '${DATAVOLSIZE}'"
 
 # RAM size is expected in MB
-RAMSIZE=$(( ${RAMSIZE} * 1024 ))
+RAMSIZE=$(( RAMSIZE * 1024 ))
 # Calculate system root partition size (1GB is allocated to the boot partition)
-SYSROOTSIZE=$(( ${DISKSIZE} - 1 - ${SWAPSIZE} - ${DATAVOLSIZE} ))
+SYSROOTSIZE=$(( DISKSIZE - 1 - SWAPSIZE - DATAVOLSIZE ))
 # System root size is expected in MB
-SYSROOTSIZE=$(( ${SYSROOTSIZE} * 1024 ))
+SYSROOTSIZE=$(( SYSROOTSIZE * 1024 ))
 # Swap size is expected in MB
-SWAPSIZE=$(( ${SWAPSIZE} * 1024 ))
+SWAPSIZE=$(( SWAPSIZE * 1024 ))
 
-KICKSTART_FILE=$(mktemp /tmp/kickstart-${VMNAME}-XXXXX.ks)
-cat ${ROOTDIR}/config/kickstart.ks.template | \
+KICKSTART_FILE=$(mktemp "/tmp/kickstart-${VMNAME}-XXXXX.ks")
+cat < "${ROOTDIR}/config/kickstart.ks.template" | \
```

Comment:
I think, in future spare time, we could edit this to use `envsubst` :)  (not in scope of this PR!)

Reply:
Oh, I do like that, and it would be easy to do here. Only question is do we want to export the VNAME,SWAPSIZE,SYSROOTSIZE or just pass them in as vars?

```
REPLACE_HOST_NAME=${VMNAME} \
REPLACE_SWAP_SIZE=${SWAPSIZE} \
REPLACE_LVM_SYSROOT_SIZE=${SYSROOTSIZE} \
envsubst < "${ROOTDIR}/config/kickstart.ks.template" > "${KICKSTART_FILE}"
```

Reply:
Can we keep to styling changes only, please? This PR is complex enough to introduce more changes.

Reply:
Agreed, I'll make a note of this and we'll change it later